modules/nw-udn-benefits.adoc

```diff
@@ -29,4 +29,9 @@ User-defined networks provide the following benefits:
 +
 * **Network parity**: With user-defined networking, the migration of applications from OpenStack to {product-title} is simplified by providing similar network isolation and configuration options.
 
-Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is: create a namespace, create and configure the custom resource, create pods in the namespace.
\ No newline at end of file
+Developers and administrators can create a user-defined network that is namespace scoped using the custom resource. An overview of the process is as follows:
++
+. An administrator creates a namespace for a user-defined network with the `k8s.ovn.org/primary-user-defined-network` label. 
+. Optional: An administrator grants a non-administrator user `admin` privileges on the user-defined network namespace.
+. The `UserDefinedNetwork` CR is created by either the cluster administrator or the user.
+. The cluster administrator or user with `admin` privileges creates pods in the namespace.
```

Comment:
nit: I would avoid saying "admin or user" and say just "user", anything the user can do admin can do.
Please ignore this comment if its a convention thing in OCP docs.